app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from typing import Any
+
+
+class Animal:
+    alive = []
```

Comment:
provide typing 
HINT: from __future__ import annotations
alive is List[Animal]

Reply:
why is it better to write this way? what is the usefulness of such writing?